fc/jira/task.py

```diff
@@ -46,6 +49,14 @@ def type_str(self) -> str:
         else:
             return 'Triage'
 
+    def triage_search(self) -> dict:
```

Comment:
Hmmm, I think we need to do something different here.  The `Task` class, for the most part, is meant to only understand _one_ task, not multiple ones.  While yes, we can shoehorn it in, I don't think that's the best way to go about it.